styles/sass/partials/_standard-button.scss

```diff
@@ -1,6 +1,13 @@
 @import "modules/transition-effects";
 @import "modules/base";
 
+%button,
+.button,
+[type=submit] {
+    // excluding <button>
+    min-width: 150px;
```

Comment:
We'll have to remember to overwritte min-width if we ever want a smaller button. But I think that's ok.